vignettes/proofr.Rmd

```diff
@@ -18,7 +18,7 @@ library(proofr)
 
 ## Authenticate with the PROOF API
 
-Run the function `proof_authenticate()`, which calls the PROOF API with your username and password, and returns an API token (an alphanumeric string).
+Run the function `proof_authenticate()`, which calls the PROOF API with your username and password, and returns an API token (an alphanumeric string). A new token is created when a new PROOF server is created, thus this API token will need to be reset.  
```

Comment:
This is not what happens. The token does not change. We can/should discuss whether it should, but currently it does not.

Reply:
@vortexing can you undo this text change, or change the wording to explain that the same token is returned when calling `proof_authenticate`